channeldb/payments.go

```diff
@@ -105,6 +105,10 @@ const (
 	// or the final cltv delta or amount is incorrect.
 	FailureReasonPaymentDetails FailureReason = 3
 
+	// FailureReasonInsufficientBalance indicates that we didn't have enough
+	// balance to complete the payment.
+	FailureReasonInsufficientBalance = 4
```

Comment:
Should add explicit `FailureReason` type.

Reply:
Added